### PR TITLE
Update configuration

### PR DESCRIPTION
**Pull request by frozenfoxx**
Expanded explanations of configuration variables. In #45 I noticed a need to update the documentation and I think this configuration page will better explain the privilege levels.

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 NEKO_CERT=              // (SSL) Cert, needed for clipboard sync
-```
\ No newline at end of file
+NEKO_KEY=               // (SSL) Key, needed for clipboard sync
+NEKO_PASSWORD=neko      // Password
+```
+
+## Environment Variables
+
+* **NEKO_ADMIN**:  the administrative user password. Any user authenticating with this will be given admin privileges.
```
Comment (nurdism) [inline docs/configuration.md]: everything looks good, but recently changed this to `NEKO_PASSWORD_ADMIN`

**Review by nurdism — CHANGES_REQUESTED** [review]

File: docs/configuration.md (hunk 1 above)
Comment (frozenfoxx) [reply docs/configuration.md]: Done, updated doc to reflect that change.

**Comment by frozenfoxx** [comment]
Just wanted to note, it may be NEKO_PASSWORD_ADMIN in the code but at least when I'm pulling the `firefox` container that's been pushed to Docker Hub only `NEKO_ADMIN` is currently working.

**Comment by nurdism** [comment]
closing in favor of https://github.com/nurdism/neko/pull/63

**Pull request closed without merging**